Tools/msi/purge.py

```diff
@@ -12,7 +12,7 @@
 
 from urllib.request import *
 
-VERSION_RE = re.compile(r'(\d+\.\d+\.\d+)(\w+\d+)?$')
+VERSION_RE = re.compile(r'(\d+\.\d+\.\d+)([A-Za-z_]+\d+)$')
```

Comment:
I'm not sure if this will affect anything, but this misses out the `?` to indicate an optional capturing group from the original pattern.

```suggestion
VERSION_RE = re.compile(r'(\d+\.\d+\.\d+)([A-Za-z_]+\d+)?$')
```